spec/controllers/idv/in_person/state_id_controller_spec.rb

```diff
@@ -1,6 +1,7 @@
 require 'rails_helper'
 
-RSpec.describe Idv::InPerson::StateIdController, allowed_extra_analytics: [:*] do
+RSpec.describe Idv::InPerson::StateIdController,
+               allowed_extra_analytics: [:sample_bucket1, :sample_bucket2] do
   include FlowPolicyHelper
   include InPersonHelper
```

Comment:
We have FSM version of State ID around still.  Should be out soon but I would update `spec/features/idv/steps/in_person/state_id_step_spec.rb` also

Reply:
This pull request was aimed at `spec/controllers` specifically, though to your point, I would hope that feature specs would flag unnecessary `allowed_extra_analytics` if we have them fully documented. And if they're not, that would imply to me we don't have good coverage in the controller specs for those controllers?

Reply:
I split off the work to address what you raised, at #11223